BEGINNER_GUIDE.md

```diff
@@ -0,0 +1,167 @@
+## Beginner's Guide
+
+First of all, we welcome you to the open-source contribution.
+
+This guide will help you to make your first contribution to the AnitaB.org.
+
+Read the project's README.md file and get a basic idea of what the project is about.
+
+Read the [Contributing guidelines](https://github.com/anitab-org/vms/blob/develop/CONTRIBUTING.md) and [Reporting guidelines](http://systers.io/reporting-guidelines). These guidelines need to be kept in mind while contributing to any project.
+---
+
+### Getting your first issue assigned
+Now since you have chosen your project, follow the steps mentioned below to get your first issue assigned.
+
+**Step 1:** Go to the **Issues**  tab of the selected project repository.
+
+![issue_tab](https://user-images.githubusercontent.com/64194946/93016427-b36c8580-f5de-11ea-82ee-57111edad230.png)
+
+
+**Step 2:** Now, from the list of the issues, search for the issue that you are comfortable working upon. Make sure that this issue is not assigned to someone else.
```

Comment:
I would suggest 
Now from the ... search for issues with label `First Timers Only` and `Status Available` for a great start! (add link to issues with both labels applied)
Remove content from make...else
The only aim of having these 2 labels is to make it easier while finding issues for newcomers and it's more easy to notice lables